packages/devtools/src/npm/index.ts

```diff
@@ -1,41 +1,31 @@
-import { getNuxtVersion } from '@nuxt/kit'
-import type { Nuxt } from 'nuxt/schema'
+import { useNuxt } from '@nuxt/kit'
 import { readPackageJSON } from 'pkg-types'
 import semver from 'semver'
-import { version as devToolsVersion } from '../../package.json'
-import type { UpdateInfo } from '../types'
+import { getPackageInfo } from 'local-pkg'
```

Comment:
Would it be helpful if we expose such utility from nypm? (i also probably need it also for both CLI and layer handling dep-resolution)

Reply:
I'd be happy to! One caveat is that in `local-pkg`, I manually wrote the code for CJS and ESM separately, as the resolution algorithms are a bit different. Which means it can not be bundled. Not sure if we could find a better isomorphic solution, otherwise are you considering this acceptable?

Reply:
Sure. Let me check local-pkg and be back to you. Have you tried mlly.resolvePath with `conditions`?